src/features/dependencies/components/DependenciesItem.tsx

```diff
@@ -44,21 +44,25 @@ const BaseDependenciesItem = ({
         <Typography
           sx={{
             color: "#333",
-            width: isEditMode ? "140px" : "auto",
-            marginLeft: isEditMode ? "20px" : "0px",
```

Comment:
These alignment properties are no longer needed because the table layout provides the needed alignments.